plugins/genesys/fu-genesys-usbhub-device.c

```diff
@@ -678,6 +682,45 @@ fu_genesys_tsdigit_value(gchar c)
 		return c - 'a' + 10;
 	return g_ascii_digit_value(c);
 }
+
+static gboolean
+fu_genesys_usbhub_device_get_bonding_and_flash_dump_location_bit(FuGenesysUsbhubDevice *self, GError **error)
+{
+	gint bonding;
+
+	if (self->dynamic_ts.bonding == 0)
+		return TRUE;
+
+	if (self->chip.model == ISP_MODEL_HUB_GL3590) {
+		self->bonding = (self->dynamic_ts.bonding & GL3590_BONDING_VALID_BIT);
+		if (self->dynamic_ts.bonding & GL3590_BONDING_FLASH_DUMP_LOCATION_BIT)
+			self->flash_dump_location_bit = TRUE;
+
+		/* success */
+		return TRUE;
+	}
+
+	/* assuming GL3521 and GL3523 use the same bonding */
```

Comment:
@adamgene I am assuming that GL3521 and GL3523 use the same bonding... however, I am pretty sure it is wrong. Tell me once you have more information. Thanks.

Reply:
As you see: the Static Tool String version is `0`,  they do not support bonding. 

https://github.com/fwupd/fwupd/blob/91e9f359c34d3fcc9fd4363e05fe70b6ab5b8b3a/plugins/genesys/fu-genesys-usbhub-device.c#L58-L67

Reply:
totally make sense.